api/app/Http/Controllers/Api/AuthController.php

```diff
@@ -16,10 +20,11 @@ class AuthController extends Controller
 {
     private StatefulGuard $guard;
 
```

Comment:
You're missing this: 

```suggestion
private EntityManager $em;
```